app/assets/javascripts/common/toggleScrollToPosition.js

```diff
@@ -0,0 +1,31 @@
+// Check for scroll position and scroll
+window.addEventListener("DOMContentLoaded", (event) => {
+  if (doesYPositionExistInLocalStorage()) {
+    scrollToYPosition();
+    removeYPositionLocalStorage();
+  }
```

Comment:
Consider inlining these functions.

Reasoning:
* These functions aren't reused anywhere else yet.
* They aren't abstracting away implementation details in a helpful way, IMO. Especially if they are just called for side effects, I'd like to see what the side effect is without having to look for its definition.
* They are short one-liners, so easy to scan even when inlined.